rq/cli/cli.py

```diff
@@ -180,6 +180,7 @@ def info(cli_config, interval, raw, only_queues, only_workers, by_queue, queues,
 @click.option('--results-ttl', type=int, default=DEFAULT_RESULT_TTL , help='Default results timeout to be used')
 @click.option('--worker-ttl', type=int, default=DEFAULT_WORKER_TTL , help='Default worker timeout to be used')
 @click.option('--job-monitoring-interval', type=int, default=DEFAULT_JOB_MONITORING_INTERVAL , help='Default job monitoring interval to be used')
+@click.option('--disable-job-desc-logging', is_flag=True, help='Toggle description logging in worker.')
```

Comment:
Can you change the help text to `Turn off description logging`? I don't think `Toggle` is a suitable term in this particular context.